apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ExplanationOfBenefitE2EBase.java

```diff
@@ -626,6 +618,34 @@ public void testEobByPatientIdForNonSamhsaEobsWithExcludeSamhsaTrueExpectNoError
         .get(requestString);
   }
 
+  /**
+   * Verifies that EOB search by patient id does not filter SAMHSA results when excludeSAMHSA is not
+   * set, as we default to false. The server call takes a long time with the amount of exhaustive
+   * SAMHSA data, so the explicit tests to test the setting set to false and the default were
+   * combined for the sake of saving 25 seconds of test execution time.
+   */
+  @Test
+  public void testEobByPatientIdWithExcludeSamhsaFalseByDefaultAndExpectNoFiltering() {
+
+    // Adjust the sampleA data that was loaded to include some samhsa data by passing true
+    String patientId = testUtils.getPatientId(testUtils.loadSampleASamhsaData());
+    String requestString = eobEndpoint + "?patient=" + patientId;
+    // This number will change if we update any the SAMHSA filter lists
+    // Pipeline loads 4437 (1 for bene, 5 for bene history) meaning the rest are claims to return
+    int numSamhsaClaims = 4431;
```

Comment:
Where does this 4,437 come from?  Can it be computed in case the source data files change and break the tests in the future?

Reply:
It's the number of records loaded by pipeline, which matches the (# of claim types * number of codes related to each) + 5. (the 5 are 1 bene, 4 bene history). I could maybe grep the pipeline total load number - 5 to get this dynamically.

Reply:
OK, just a maintenance issue.  Not urgent to automate the number if that would delay the PR.  We can revisit that in a future sprint.

Reply:
Will followup in https://jira.cms.gov/browse/BFD-3168